pilot/pkg/networking/plugin/authz/rbac_test.go

```diff
@@ -1087,10 +1110,20 @@ func TestConvertRbacRulesToFilterConfigPermissive(t *testing.T) {
 			},
 			expectConfig: emptyConfig,
 		},
+		{
+			name:    "global permissive",
+			service: &serviceMetadata{name: "service"},
+			option: rbacOption{
+				roles:    roles,
+				bindings: []model.Config{},
```

Comment:
Could you add an actual binding here, so that it can cover the case that when globalPermissive is true, any normal rules is not included. Otherwise, if the bindings is empty, there won't be any normal rules even if  `globalPermissive` is false.

Reply:
updated test case.